### PR TITLE
Introduce passwords section in the config

### PR DESCRIPTION
**Pull request by PremyslCerny**

### CODE REVIEW

**Comment by PremyslCerny** [comment]
@ateska Please see.

File: asab/api/service.py [hunk 1]
```diff
@@ -40,4 +40,4 @@ def _initialize_web(self, app, listen):
 
 
 	async def config(self, request):
-		return asab.web.rest.json_response(request, {s: dict(asab.Config.items(s)) for s in asab.Config.sections()})
+		return asab.web.rest.json_response(request, asab.Config.get_items_without_passwords())
```
Comment (ateska) [inline asab/api/service.py]: Nope.
```
res = asab.Config.copy()
del res['passwords'] # Or better, replace values of the password items with '***' 
return res
```

File: asab/config.py [hunk 2]
```diff
@@ -157,8 +172,39 @@ def _load(self):
 
 		self.add_defaults(ConfigParser._default_values)
 
+		# Obtain password section
+		self._passwords = dict(self.items("passwords"))
+
+		# The password section is truly loaded after all other sections are interpolated
+		# Iterate through all configuration options and parse passwords
+		for section in self._sections.keys():
```
Comment (ateska) [inline asab/config.py]: I'm actually not sure about this automated expansion ... I would pass that to the individual user (e.g. MySql connection will go to the passwords itself).
Or there is some kind of expansion in Python ConfigParser already ... and I propose to use that one then.

This doesn't seems right ...

**Comment by PremyslCerny** [comment]
@ateska Please see again.

**Comment by PremyslCerny** [comment]
@margirova Please check.

**Comment by margirova** [comment]
There are no triple for-loops, I am happy:)

**Comment by PremyslCerny** [comment]
@ateska Please merge! Approved by @margirova.

**Comment by PremyslCerny** [comment]
@margirova There are, but hidden. That's what lovely about Python. :)

**Pull request merged**